src/docs/cookbook/animation/page-route-animation.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Animate a page route transition
+next:
+  title: Animate the properties of a container
+  path: /docs/cookbook/animation/animated-container
+---
+
+# Animate a page route transition
+
+A design language, such as Material, defines standard behaviors when
+transitioning between routes (or screens). Sometimes, though, a custom
+transition between screens can make an app more unique. To help,
+[PageRouteBuilder](https://api.flutter.dev/flutter/widgets/PageRouteBuilder-class.html)
```

Comment:
Generally, we use the following format for links. (We didn’t always do that, but it’s now part of our recommended guidelines.) Also, we use macros for the common websites. So:

To help, [PageRouteBuilder] provides an [Animation]…

[PageRouteBuilder]:  {{site.api}}/flutter/widgets/PageRouteBuilder-class.html
[Animation]: {{site.api}}/flutter/widgets/PageRouteBuilder-class.html

It makes the text more readable and it makes things a lot easier if we change the URLs for our API website, for example. (Which we did last year.)
https://github.com/dart-lang/site-shared/wiki/Writing-for-Dart-and-Flutter-websites

Reply:
👍 fixed